### PR TITLE
Issue #4280

### PR DESCRIPTION
**Pull request by ankitkataria**
Hey, 
I was trying to solve issue 4280
- I added the required page_white_put icon
- i realised that the themes of OBS is managed in a separate repo and update_gento.sh copied all the required assets(images and icons) from the project, so i added a line to copy the icon, and ran the script. - I made changes to icons.scss and _files_view.html.erb. **BUT** the icon doesn't still seem to work i still get the old icon. 
- I figured that OBS is using image spriting, i tried reading the file /src/api/lib/tasks/sprites.rake, i figured this is the file that generates the sprited image, but i couldn't figure out when it is called, and could i call it to make a new sprited image with the newly added icon.
Any help?

### CODE REVIEW

**Comment by ankitkataria** [comment]
@Ana06 Any help?

**Comment by bgeuken** [comment]
Hi @ankitkataria, first of all thanks for your contribution:bouquet:

It looks like you did not update the [sprite file](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/assets/images/icons_sprite.png). Just run the script;-)
This will also update the [icons.scss](assets/stylesheets/webui/application/icons.scss) properly.

File: src/api/app/assets/stylesheets/webui/application/icons.scss [hunk 1]
```diff
@@ -120,6 +120,7 @@
 .icons-page_white_add { width: 16px; height: 16px; background: image-url('icons_sprite.png') -342px -400px no-repeat; }
 .icons-page_white_delete { width: 16px; height: 16px; background: image-url('icons_sprite.png') -362px -400px no-repeat; }
 .icons-page_white_get { width: 16px; height: 16px; background: image-url('icons_sprite.png') -382px -400px no-repeat; }
+.icons-page_white_put { width: 16px; height: 16px; background: image-url('icons_sprite.png') -382px -400px no-repeat; }
```
Comment (bgeuken) [inline src/api/app/assets/stylesheets/webui/application/icons.scss]: Note the px coordinates (eg. ` -382px -400px `) at the end of each line. Each of them points to a single icon on the [sprite file](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/assets/images/icons_sprite.png).

**Comment by bgeuken** [comment]
@ankitkataria and please add a short explanation to the commit message why you add the page_white_get icon.

Thanks:-)

**Comment by ankitkataria** [comment]
image
fixed the download icon, what all i did:
- changed update_bento.sh to also copy the page_white_put icon from [opensuse themes](https://github.com/openSUSE/opensuse-themes) to the required assets/icons/ directory
- changed _files_view.html.erb 
- ran the assets:reprite task to produce the newly sprited images and update icons.scss

thankyou both for the help and support. :smile:

**Comment by bgeuken** [comment]
@ankitkataria Regarding the comment, what @Ana06 meant was to add this to the bottom of the commit:

`
Fixes #4280
`

That way GitHub automatically closes the issue when the PR get's merged.

**Comment by ankitkataria** [comment]
thankyou @bgeuken , i will take take of the commit messages from the next time. :smile:

**Comment by Ana06** [comment]
@bgeuken 

> @ankitkataria Regarding the comment, what @Ana06 meant was to add this to the bottom of the commit:
> 
> Fixes #4280
> 
> That way GitHub automatically closes the issue when the PR get's merged.

Yes! Do not add the issue to the title 🙈 The title should be something short that allow you know what the commit is about, and nobody remember issues numbers 😉 So, just wiriting `Fixes #4280` at the end of the description is enough. 👍

**Comment by bgeuken** [comment]
@ankitkataria @Ana06 I noticed, but one thing at a time;-)

**Pull request merged**